### ISSUE TITLE
Doc links no longer work without the .html extension

### ISSUE THREAD

**Issue by hegemonic** [googleapis/nodejs-spanner#662]
As recently as one or two days ago, you could link to the Node.js API docs without including the `.html` file extension. For example, this link worked: https://googleapis.dev/nodejs/spanner/latest/Table

Today, opening that link results in a 404, and you have to include the `.html` extension: https://googleapis.dev/nodejs/spanner/latest/Table.html

This change breaks a number of links to the API docs from cloud.google.com.

@bcoe FYI

**Comment by bcoe** [comment]
@hegemonic we have never supported the `.html` extension on `googleapis.dev` (I believe it was a nicety introduced by the URL rendering on cloud.google.com).

A regex is used to append a `.html` onto any historical links on `cloud.google.com`, see:

[https://cloud.google.com/nodejs/docs/reference/spanner/latest/Table](https://cloud.google.com/nodejs/docs/reference/spanner/latest/Table)

My guess is that there was a link added to cloud.google.com that's simply missing the `.html`; the googleapis.dev service currently runs no intermediate proxy, serving directly out of a storage bucket, tldr; it doesn't currently support any URL rewriting.

**Comment by hegemonic** [comment]
To clarify: Links to `googleapis.dev` were definitely working without the `.html` extension at the beginning of this week, however unlikely that might sound. I tested those links a week or two ago, when I updated the Cloud Spanner docs to point to `googleapis.dev` instead of `cloud.google.com`, and the `cloud.google.com` link checker didn't start complaining about those links until shortly before I filed this bug.

But it sounds like that was just a happy accident, or it was caused by cosmic rays, or who knows what, not the intended behavior. I've updated the links on `cloud.google.com` already, so I'll close this bug.
